### DISCUSSION TITLE
Unable to access workbench menus

### DISCUSSION THREAD

**Discussion by markswanusa** [BlitterStudio/amiberry#1307] [Q&A] [answered]
On a M2 MacBook Air ...

After using FS-UAE for several years, and seeing that FS-UAE has not gotten updates for a long time, I decided to switch to AmiBerry.  One of the things I just cannot get working is how to access the Workbench menus.  In FS-UAE, I could hold down the control key while pressing the trackpad and that would access the Workbench menus.  I have tried all the obvious things like Shift, Control, Option, Command and, seemingly, all multi-key combinations.  I do not currently have a mechanical mouse to plug into my Air to see if those buttons are recognized.  If that turns out to be the only way to access the Workbench menus, then I will get one.

**Comment by midwan** [comment, marked as answer]
I'm afraid there's no special emulator shortcut to emulate the right click that you're after, which I'm assuming FS-UAE had. It's certainly possible to implement something, if the requirement is there (but so far, nobody else has even mentioned this). 

The Amiga-side shortcut for right click would be to use RAmiga+RAlt, but I'm not sure if you've got the equivalent keys on your MacBook. Having an external mouse definitely seems like the best solution to this.

**Comment by markswanusa** [comment]
Thanks for the reply.
